frontend/public/components/environment.jsx

```diff
@@ -406,10 +406,18 @@ export class UnconnectedEnvironmentPage extends PromiseComponent {
     const { currentEnvVars, containerType } = this.state;
     const currentEnv = _.cloneDeep(currentEnvVars);
     currentEnv.setFormattedVars(containerType, i, type, env.nameValuePairs);
+
+    const emptyNamesWithValue = _.filter(
+      env.nameValuePairs,
+      (pair) => pair[NameValueEditorPair.Name] === '' && pair[NameValueEditorPair.Value].length,
+    );
+    const _hasEmptyName = emptyNamesWithValue.length > 0;
```

Comment:
Hey @yozaam big thanks for your PR! :+1: 

Just for your info, we try to avoid lodash in new code, but I think its fine here because the component already uses lodash.

If you use lodash you can also use `_.some` which returns directly a boolean if some of the array items matches your criteria. Then you can assign it directly to your variable and don't need the length condtion.

Please also remove the `_` prefix from your variable, we normally don't prefix/suffix them. This allows you to set the state like this:

```tsx
    this.setState({
      currentEnvVars: currentEnv,
      dirty: true,
      success: null,
      hasEmptyName,
    });
```

The `hasEmptyName`  at the end is just a short form for `hasEmptyName: hasEmptyName`.

And then something else: Did you tested your code also with the Add ConfigMap/Secrets option? I think its enough to check if the name is empty. An empty environment variable value should be allowed.

Reply:
> I think its enough to check if the name is empty. An empty environment variable value should be allowed.

Backend doesn't give an error if `Name is empty and Value is empty`, 
so I thought I needed to check only `if Name is empty and Value is **_not_** empty` 

E.g. this doesn't give an error in Backend
![image](https://user-images.githubusercontent.com/20089340/130783825-b9867eee-0302-4a72-8d0c-ba8a0751da74.png)

Should I prevent this and check only name?